src/sentry/api/utils.py

```diff
@@ -498,10 +498,10 @@ def id_or_slug_path_params_enabled(
         return False
 
     # EA option for endpoints where organization is not available
-    if convert_args_class in options.get("api.id-or-slug-enabled-ea-endpoints"):
-        return True
+    if convert_args_class:
+        return convert_args_class in options.get("api.id-or-slug-enabled-ea-endpoints")
 
-    return False
+    return True
```

Comment:
Is this correct? If this function is called with no parameters should it return true?

Reply:
If `convert_args_class` is not passed in,  but the org is passed in and in the org option, which is the case ` get_region_for_organization`, I want to return True. All the other uses of this wrapper pass in the convert_args _class.

Reply:
I am going to pull this logic and add an other option.